### PR TITLE
[RFR] Fix jenkins chrome crash issue

### PR DESCRIPTION
**Pull request by stillalearner**
This PR solves the jenkins chrome crash issue by separating the analysis file into two files.

### CODE REVIEW

File: cypress/e2e/models/projects.ts [hunk 1]
```diff
@@ -120,6 +120,7 @@ export class Projects {
         shouldBeEnabled(primaryButton, next);
         cy.wait(10 * SEC);
         clickByText(primaryButton, next);
+        clickByText(primaryButton, next);
```
Comment (sshveta) [inline cypress/e2e/models/projects.ts:123]: why are we clicking twice ?

File: cypress/e2e/tests/analysis.test.ts [hunk 2]
```diff
@@ -7,96 +7,14 @@ import { Projects } from "../models/projects";
 import { completed } from "../types/constants";
 import { skipOn } from "@cypress/skip-test";
 
-describe(["tier1"], "Analysis", function () {
+describe(["tier1"], "Analysis2", function () {
```
Comment (sshveta) [inline cypress/e2e/tests/analysis.test.ts:10]: Why Anlaysis 2?

We can name this standard analysis tests

File: cypress/e2e/tests/targets.test.ts [hunk 3]
```diff
@@ -0,0 +1,105 @@
+/// <reference types="Cypress" />
+/// <reference types='cypress-tags' />
+
+import { getRandomApplicationData, isInstalledOnOCP, login, trimAppNames } from "../../utils/utils";
+import { Analysis } from "../models/analysis";
+import { Projects } from "../models/projects";
+import { completed } from "../types/constants";
+import { skipOn } from "@cypress/skip-test";
+
+describe(["tier1"], "Analysis", function () {
```
Comment (sshveta) [inline cypress/e2e/tests/targets.test.ts:10]: And this one we can name 
"analysis on different targets"

**Review by sshveta — CHANGES_REQUESTED** [review]

File: cypress/e2e/tests/analysis.test.ts (hunk 2 above)
Comment (stillalearner) [reply cypress/e2e/tests/analysis.test.ts:10]: done

File: cypress/e2e/tests/targets.test.ts (hunk 3 above)
Comment (stillalearner) [reply cypress/e2e/tests/targets.test.ts:10]: done

**Review by sshveta — APPROVED** [review]

**Pull request merged**